### ISSUE TITLE
Impaired toe-walking ability

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7167]
**Preferred term label:**
Impaired toe-walking ability

**Synonyms**

**Definition (free text, please give PubMed ID)**
Difficulty or reduced ability to walk on toes. Toe-walking can be tested as a part of the neurological examination. Foot plantar flexion weakness leads to difficulties in walking on toes.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
